src/plugins/featureFlags.ts

```diff
@@ -16,7 +16,7 @@ const defaultFlagSet: LDFlagSet = {
   'banner-text': '', // by default, there is no banner text
   'enable-web-chat': false, // by default, old webchat is disabled
   'enable-genesys-web-message': false, // by default, genesys web message is disabled
-  'is-support-society': true // by default support society
+  'enable-society': true // by default society is enabled
```

Comment:
The default (or fallback) value should disable any new features, so that it's safe for prod users in case LaunchDarkly is not available (or in case they block LauchDarkly -- we don't want them to see new features by default).